vapi_python/vapi_python.py

```diff
@@ -62,3 +62,21 @@ def start(
     def stop(self):
         self.__client.leave()
         self.__client = None
+
+    def send(self, message_type, message_content):
```

Comment:
could you make the interface more generic `send(self, message)`

here's the behaviour from [web SDK](https://github.com/VapiAI/web/blob/cb7f315e49889c84e1b03bfaa1664d14d6b62608/vapi.ts#L319C2-L321C4)

```
 send(message: VapiClientToServerMessage): void
 ```

Reply:
Thanks. Made the changes. I guess ,we can add the say and control functions later as I am not sure how to test them correctly :)